### DISCUSSION TITLE
How to enable translation for api calls

### DISCUSSION THREAD

**Discussion by shihuojian** [lllyasviel/Fooocus#2673] [Q&A]
Hello! How to enable the translation function by default in the config configuration? I don’t see the translation configuration in the config

**Comment by mashb1t** [comment]
Which translation are you referring to, the UI translation from english to another language?
Fooocus doesn't currently offer a prompt translation feature out of the box, but there are 2 PRs for it, inclm one from me using Google Translate.

**Reply by shihuojian** [reply]
https://github.com/mashb1t/Fooocus is in this branch
The function of automatically translating prompt words into English needs to be set by default in the configuration file.

**Reply by mashb1t** [reply]
This is a fork, not a branch. This discussion should have been opened in my repo.
No matter the fact, there currently isn't an option to enable translation by defaulty but you have to modify the default value in https://github.com/mashb1t/Fooocus/blob/main/webui.py#L336 from False to True.

**Reply by shihuojian** [reply]
Thank you very much. I opened it in mashb1t/Fooocus, but it will automatically jump here~
